cocos/dragon-bones/ArmatureCache.ts

```diff
@@ -419,20 +464,29 @@ export class AnimationCache {
         }
     }
 }
-
+/**
+ * @en Cached data of armature.
+ * @zh 骨架缓存。
+ */
 export class ArmatureCache {
     protected _privateMode = false;
     protected _animationPool: Record<string, AnimationCache> = {};
     protected _armatureCache: Record<string, ArmatureInfo> = {};
 
     constructor () {
     }
-
+    /**
+     * @en Enable private cache mode.
+     * @zh 启用私有缓存模式。
+     */
     enablePrivateMode () {
         this._privateMode = true;
     }
 
-    // If cache is private, cache will be destroy when dragonbones node destroy.
+    /**
+     * @en If cache is private, cache will be destroy when dragonbones node destroy.
```

Comment:
If using private cache mode, all cached data will be destroyed when corresponding dragonbone nodes are destroyed.